modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/LiveSchemaChangeTableTest.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.runner.app;
+
+import java.util.List;
+import java.util.UUID;
+import org.apache.ignite.app.Ignite;
+import org.apache.ignite.internal.schema.SchemaType;
+import org.apache.ignite.internal.table.ColumnNotFoundException;
+import org.apache.ignite.internal.table.TableImpl;
+import org.apache.ignite.table.KeyValueBinaryView;
+import org.apache.ignite.table.Table;
+import org.apache.ignite.table.Tuple;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Live schema tests.
+ */
+@Disabled("https://issues.apache.org/jira/browse/IGNITE-14581")
+class LiveSchemaChangeTableTest extends AbstractSchemaChangeTest {
+    /**
+     * Check live schema add columns
+     */
+    @Test
+    public void testLiveSchemaAddColumns() {
+        List<Ignite> grid = startGrid();
+
+        createTable(grid);
+
+        Table tbl = grid.get(1).tables().table(TABLE);
+
+        ((TableImpl)tbl).schemaType(SchemaType.LIVE_SCHEMA);
+
+        Tuple row = tbl.tupleBuilder().set("key", 1L).set("valStrNew", "111").set("valIntNew", 333).build();
+
+        tbl.insert(row);
```

Comment:
Looks like both "inserts" cheks the same.
Let's check
1. a row of old schema can be successfully updated with row of new schema.
2. a row of new schema can be successfully inserted.
3. Inserting row of old schema will not lead to column removal.

Reply:
I have divided test into 3 parts to cover all the cases.